ClosedXML/Excel/Ranges/IXLRangeBase.cs

```diff
@@ -118,20 +109,18 @@ public interface IXLRangeBase : IXLAddressable
         IXLCell FirstCellUsed();
 
         /// <summary>
-        ///   Returns the first cell with a value of this range.
+        /// Returns the last cell with a value of this range.
```

Comment:
Looks like this should be `first`
```suggestion
        /// Returns the first cell with a value of this range.
```

Reply:
Also, the comment may be a bit misleading - cells considered non-empty does not necessarily have a value. Maybe 
```suggestion
        /// Returns the first non-empty cell of this range.
```
?